sharktank/sharktank/types/tensors.py

```diff
@@ -1190,6 +1232,8 @@ def create(
         extra_properties: dict[str, Any],
     ) -> "InferenceTensor":
         shard_count = int(extra_properties["shard_count"])
+        devices = extra_properties["devices"]
```

Comment:
Same here.

Reply:
If we don't save devices, how should we handle the for-loop setting DeviceTensorTrait below?